modules/netaddress.go

```diff
@@ -59,6 +59,10 @@ func (na NetAddress) isLoopback() bool {
 	}
 	return false
 }
+// exported form of isLoopback
+func (na NetAddress) IsLoopback() bool {
+	return na.isLoopback()
+}
```

Comment:
The proper thing to do here is to replace `isLoopback` with `IsLoopback`. There are very few instances that you need to change, so no excuses.